frontend/src/concepts/connectionTypes/fields/DropdownFormField.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import { Badge, MenuToggle, Select, SelectList, SelectOption } from '@patternfly/react-core';
+import { DropdownField } from '~/concepts/connectionTypes/types';
+import DataFormFieldGroup from '~/concepts/connectionTypes/fields/DataFormFieldGroup';
+
+type Props = {
+  field: DropdownField;
+  isPreview?: boolean;
+  onChange?: (value: string[]) => void;
+  value?: string[];
+};
+
+const DropdownFormField: React.FC<Props> = ({ field, isPreview, onChange, value }) => {
```

Comment:
Nit: Calling it a Dropdown but using a Select could be confusing.

Reply:
That's fair. It coincides with the type. And the type name will be shown to the user as an available type.

Perhaps we should discuss the type name with us?